.gitignore

```diff
@@ -87,8 +87,6 @@ celerybeat-schedule
 *.sage.py
 
 # Environments
-.env
-.venv
```

Comment:
Perfect, but now you can put `.env` back I think (in a second commit). With this commit, everyone will get the file. In the future, we do not want people committing changes to it. It should be a local offline file

Reply:
Yep, the purpose of this was that everyone could get the .env file.